pkg/velaql/providers/query/tree.go

```diff
@@ -215,6 +215,15 @@ func init() {
 			}),
 			GroupResourceType: GroupResourceType{Group: "batch", Kind: "CronJob"},
 		},
+		ChildrenResourcesRule{
+			GroupResourceType: GroupResourceType{Group: "batch", Kind: "Job"},
+			SubResources: buildSubResources([]*SubResourceSelector{
+				{
+					ResourceType: ResourceType{APIVersion: "v1", Kind: "Pod"},
+					listOptions:  jobLabelListOption,
```

Comment:
I think the Job resource could use the `defaultWorkloadLabelListOption`.

Reply:
Yes, my reference is wrong. Check the k8s code and find that getPodsForJob uses "spec", "selector"